src/RegularizedProblems.jl

```diff
@@ -12,6 +13,7 @@ include("lrcomp_model.jl")
 include("matrand_model.jl")
 include("group_lasso_model.jl")
 include("nnmf.jl")
+include("denoising_model.jl")
```

Comment:
Use `@require` to load the model if the required dependencies have been imported.